omnibus/files/private-chef-cookbooks/private-chef/libraries/helper.rb

```diff
@@ -286,4 +286,9 @@ def self.chef_server_running_content(node)
     }
     Chef::JSONCompat.to_json_pretty(content)
   end
+
+  def postgresql_sslmodes
+    # current driver-supported sslmodes
```

Comment:
It might be useful to specify what current means here.

Reply:
Changed to:
```
    # driver-supported sslmodes
```